lambdas/indexer/app.py

```diff
@@ -65,8 +75,17 @@ def health(keys: Optional[str] = None):
             body = health.as_json(keys)
         except RequirementError:
             raise BadRequestError('Invalid health keys')
-        return Response(body=json.dumps(body),
-                        status_code=200 if body['up'] else 503)
+    return Response(body=json.dumps(body),
+                    status_code=200 if body['up'] else 503)
+
+
+@app.schedule('rate(1 minute)', name=config.indexer_cache_health_lambda_basename)
+def generate_health_object(event: chalice.app.CloudWatchEvent):
+    assert event is not None  # avoid linter warning
```

Comment:
This should not be necessary. Can you post screen shot with the warning? I assume by "linter" you mean PyCharm, right?

Reply:
<img width="660" alt="Screen Shot 2019-09-25 at 11 18 28 AM" src="https://user-images.githubusercontent.com/19215510/65628403-3c318200-df86-11e9-8a68-c8b0b671ca1b.png">

Reply:
@hannes-ucsc 
Replicating [nudge lambda](https://github.com/DataBiosphere/azul/blob/45eb2228fe4c648cec06de27c3f30d2ee057d5b2/lambdas/indexer/app.py#L452) work.

Reply:
Well, the person who committed that must have been intellectually challenged.

I've since learned that naming the parameter with a leading underscore also suppresses that warning.